frontend/tests/playwright/product/product-list.spec.ts

```diff
@@ -0,0 +1,207 @@
+import { test, expect } from '../test-fixtures';
+import { createGraphQLHandler } from '../msw/request-handler';
+
+test.describe('Product List Data', () => {
+   test.skip(true, 'Only run once mocks to getProductList are fixed.');
```

Comment:
Is this test intentionally skipped?

Reply:
yes. there is an issue where mocks with strapi are completely busted. waiting for that to be fixed, but ive checked without the mocking to make sure the tests will pass.